### DISCUSSION TITLE
Is there a woking bypass yet

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#91] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
If there is a bypass that works please tell me it my teachers keep closeing all my tabs and shit its mad annyoing

**Comment by md4r2** [comment, marked as answer]
For the extensions I've heard a new one is being developed but in the meantime u could use a proxy site like [HU (holy unblocker)](https://website-aio.herokuapp.com/).

**Reply by ghost** [reply]
i have go guardian thats the thing and all my teachers just sit there and watch my tabs

**Reply by Alex2Mayhem** [reply]
fr gg is anoyying AF please fix

**Reply by ghost** [reply]
ong
